docs-conceptual/azps-3.8.0/new-azureps-module-az.md

```diff
@@ -1,94 +1,102 @@
 ---
 title: Introducing the Azure PowerShell Az module
 description: Introducing the new Azure PowerShell module Az, the replacement for the AzureRM module.
-ms.date: 05/10/2019
+ms.date: 05/15/2020
 ms.devlang: powershell
 ms.topic: conceptual
 ---
 # Introducing the new Azure PowerShell Az module
 
-Starting in December 2018, the Azure PowerShell Az module is in general release and is now the intended
-PowerShell module for interacting with Azure. Az offers shorter commands, improved stability, and
-cross-platform support. Az also has feature parity with AzureRM, giving you an easy migration
-path.
+Starting in December 2018, the Azure PowerShell Az module is in general release and is now the
+intended PowerShell module for interacting with Azure. Az offers shorter commands, improved
+stability, and cross-platform support. Az also has feature parity with AzureRM, giving you an easy
+migration path.
 
-With the Az module, Azure PowerShell is now compatible with PowerShell 5.1 on Windows and PowerShell Core 6.x
-and later on all supported platforms - including Windows, macOS, and Linux.
+> [!NOTE]
+> PowerShell 7.x and later is the recommended version of PowerShell for use with Azure PowerShell on
+> all platforms.
+
+With the Az module, Azure PowerShell works with PowerShell 6.2.4 and later on all platforms
```

Comment:
My feeling is this page introduce overall Azure PowerShell. I prefer to adding "latest". It becomes to "With the latest Az module ....."

Reply:
Added "latest" as suggested.